src/main/java/com/requestrealpiano/songrequest/controller/LetterController.java

```diff
@@ -25,22 +27,30 @@
 
     @ResponseStatus(HttpStatus.OK)
     @GetMapping
-    public ApiResponse<List<LetterResponse>> findAll() {
-        List<LetterResponse> letters = letterService.findAllLetters();
+    public ApiResponse<LettersResponse> findAll(@ModelAttribute @Valid PaginationParameters paginationParameters) {
+        LettersResponse letters = letterService.findAllLetters(paginationParameters);
         return SUCCESS(OK, letters);
```

Comment:
```suggestion
        return SUCCESS(OK, letterService.findAllLetters(paginationParameters));
```
- p4 : 이렇게 하면 코드를 줄일 수 있긴 한데.. 개인적인 선호라고 생각되어지네요 ㅎㅎ